packages/bitcore-lib/test/publickey.js

```diff
@@ -451,4 +451,36 @@ describe('PublicKey', function() {
 
   });
 
+  describe('#isValidTaproot', function() {
```

Comment:
Create a test which stubs fromTaproot and throws/handles an error, returning false.

Reply:
That's what the general failure does (invalid X)